src/ResourceManager/Version2021_01_01/Customized/SubscriptionExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Management.ResourceManager.Version2021_01_01.Models.Utilities
```

Comment:
I think the namespace should be `Microsoft.Azure.Commands.ResourceManager.Version2021_01_01.Customized`, just like `SubscriptionClientExtension.cs`

Reply:
To keep the consistent with the existing codes, we prefer to use Microsoft.Azure.Management.ResourceManager.Version2021_01_01.Models.Utilities.